### DISCUSSION TITLE
1.32.2

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#18489] [General]
# Improvements
- [#6010](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6010) Add new features for Onesti Products AS easyCodeTouch_v1 (@slackspace-io)
- [#6008](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6008) Expose `battery` for Visonic MP-840 (@EnsuingRequiem)
- [#5657](https://github.com/Koenkk/zigbee-herdsman-converters/pull/5657) Support `action` and `switch_mode` for TuYa TS0726 (@Koenkk)
- [#5987](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5987) Add findByModel function for easier inheritance in external converters (@kirovilya)
- [#5977](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5977) Add "Tempo" Tarif for LiXee ZLinky_TIC (@SilentT-FR)
- [#5979](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5979) Support hue and saturation for Calex 421792 (@JoyceKimberly)
- [#16574](https://github.com/Koenkk/zigbee2mqtt/issues/16574) Support sensitivity for ADEO LDSENK08 (@Koenkk)
- [#5971](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5971) Support pin and add `fingerprint` action for Heimgard Technologies HC-SLM-1 (@nagydavid)
- [#5968](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5968) Expose power outage memory for Mercator Ikuü SPP02G (@aidbish)
- [#5963](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5963) Support OTA for Legrand 067773 and 067774 (@SilentT-FR)
- [#10740](https://github.com/Koenkk/zigbee2mqtt/issues/10740) Expose `toggle` action for Philips 929003017102 (@Koenkk)
- [#5952](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5952) Update NodOn / ADEO description files and support OTA for some (@AlexisPolegato)
- [#5936](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5936) Add OTA for multiple Legrand devices (@SilentT-FR)
- [#730](https://github.com/Koenkk/zigbee-herdsman/issues/730) Support 16 bytes install codes for EZSP adapter (@NicolaiSoeborg)

# Fixes
- [#18180](https://github.com/Koenkk/zigbee2mqtt/issues/18180) Fix `'dict object' has no attribute 'ir_code_to_send'` error (@Koenkk)
- [#18372](https://github.com/Koenkk/zigbee2mqtt/issues/18372) Fix insecure Zigbee network encryption key generation (@0xbb)
- [#10740](https://github.com/Koenkk/zigbee2mqtt/issues/10740) Fix not all cluster unbound/bound when specified (@Koenkk)
- [#18388](https://github.com/Koenkk/zigbee2mqtt/issues/18388) Fix socket error crashing Zigbee2MQTT (@Koenkk)
- [#18306](https://github.com/Koenkk/zigbee2mqtt/issues/18306) Use HA sensor device class `atmospheric_pressure` instead of `pressure` (@bdr99)
- [#6020](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6020) Fix configure failing for OWON PCT504 (@XLDom)
- [#6021](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6021) Fix typo with Noes A1Z (@qp68)
- [#6011](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6011) Fix on/off reporting and disable unsupported power on behaviour for Schneider Electric S520530W (@slackspace-io)
- [#18391](https://github.com/Koenkk/zigbee2mqtt/issues/18391) Disable unsupported `power_on_behavior` for Leviton DG15S-1BW (@Koenkk)
- [#18364](https://github.com/Koenkk/zigbee2mqtt/issues/18364) Disable unsupported `power_on_behavior` for ZLED-RGB9 (@Koenkk)
- [#5970](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5970) Mark MOTDETAT as not reportable for LiXee ZLinky_TIC (@SilentT-FR)
- [#2132](https://github.com/Koenkk/zigbee2mqtt.io/pull/2132) Update some Lidl light model numbers (@Koenkk)
- [#15111](https://github.com/Koenkk/zigbee2mqtt/issues/15111) Fix `power_outage_memory` not working for Xiaomi ZNCZ12LM  (@thekev)
- [#16709](https://github.com/Koenkk/zigbee2mqtt/issues/16709) Fix invalid 0 power measurements for TuYa `_TZ3000_b28wrpvx` (@Koenkk)
- [#18238](https://github.com/Koenkk/zigbee2mqtt/issues/18238) Fix Third Reality 3RSNL02043Z occupancy detection not working (@Koenkk)
- [#5948](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5948) Fix timers of RTX ZVG1 and Saswell SAS980SWT-7-Z01 (@ScratMan)
- [#18171](https://github.com/Koenkk/zigbee2mqtt/issues/18171) Fix Namron/Lightsolutions Eco-Dim.07/Eco-Dim.10 not recognised (@Koenkk)
- [#5943](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5943) Fix no energy measurements for `_TZ3000_qeuvnohg` (will use polling now) (@clausmuus)
- [#5941](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5941) Fix QT-07S not reporting values  (@phoenixswiss)
- [#18171](https://github.com/Koenkk/zigbee2mqtt/issues/18171) Fix Eco-Dim.07/Eco-Dim.10 not detected as supported (@Koenkk)
- [#17861](https://github.com/Koenkk/zigbee2mqtt/issues/17861) Silence `Moes BHT-002: Unrecognized DP` warning message (@Koenkk)
- [#10740](https://github.com/Koenkk/zigbee2mqtt/issues/10740) Fix incorrect bind cluster shown when cluster is manufacturer specific (@Koenkk)
- [#18121](https://github.com/Koenkk/zigbee2mqtt/issues/18121) Fix ping for devices which have genBasic on non first endpoint (@Koenkk)

# New supported devices
This release adds support for 44 devices: 
- [#6018](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6018) `1444420` Heatit Zig Dim 250W (@geirra)
- [#6028](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6028) `84870058` ADEO ENKI LEXMAN Extraflat 225  (@cesarblancg)
- [#18455](https://github.com/Koenkk/zigbee2mqtt/issues/18455) `9290024406A` Philips Hue P45 light bulb (@RPiNut)
- [#6023](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6023) `H10` Ubisys Heatingcontrol 10-Way (@sjorge)
- [#5991](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5991) `SIN-4-1-22_LEX` ADEO ENKI LEXMAN Access Control (@RAULSALABIGAS)
- [#18447](https://github.com/Koenkk/zigbee2mqtt/issues/18447) `TS0203_1` TuYa Door sensor with scene switch (@AskDev2022)
- [#6019](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6019) `SOMFY-1241752` SOMFY Blinds from vendors using this roller (@nonodename)
- [#5990](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5990) `ZY-M100-S_2` TuYa Mini human breathe sensor (@arteck)
- [#6012](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6012) `915005988201` Philips Hue Gradient light tube large black EU (@tititg34)
- [#6006](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6006) `WB-MSW-ZIGBEE v.4` Sprut.device Wall-mounted Zigbee sensor (@aadegtyarev)
- [#6007](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6007) `ZYXH` TuYa 24 gang switch (@kirovilya)
- [#5998](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5998) `GB-540` Visonic Glass break detector (@gregwjacobs)
- [#6002](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6002) `QS-Zigbee-SEC01-U` TuYa Zigbee 3.0 smart light switch module 1 gang (@jh-616)
- [#6000](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6000) `YXZBSL` TuYa Smart siren (@kirovilya)
- [#5995](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5995) `10454466` HORNBACH FLAIR Viyu Smart LED GU10 RGBW lamp (@xvisor16)
- [#5989](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5989) `4033931P6` Philips Hue white ambiance suspension Fair (@clamb)
- [#5994](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5994) `929003521501` Philips Hue Go portable table lamp black (@shing6326)
- [#5994](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5994) `929003521701` Philips Hue Go portable table lamp black (@shing6326)
- [#5993](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5993) `QBKG18LM` Xiaomi Aqara T1 double rocker without neutral (@cloudyngcloudy)
- [#5992](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5992) `SPM02` TuYa Smart energy monitor for 3P+N system (@script0803)
- [#5976](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5976) `U86Z223A10-ZJU01(GD)` Honyar Smart power socket 10A with USB (with power monitoring) (@65698888)
- [#18358](https://github.com/Koenkk/zigbee2mqtt/issues/18358) `915005987801` Philips Hue white and color ambiance gradient Signe floor lamp (black) (@grahamhayes)
- [#5985](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5985) `YXZBRB58` TuYa Smart human presence sensor (@kirovilya)
- [#5978](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5978) `1402767` Namron Zigbee dimmer 2-pol 250W (@runnane)
- [#18338](https://github.com/Koenkk/zigbee2mqtt/issues/18338) `8719514491106` Philips Hue white ambiance E14 (@Koenkk)
- [#18341](https://github.com/Koenkk/zigbee2mqtt/issues/18341) `929003053401` Philips Hue Flourish white and color ambiance table light with Bluetooth (@maxk1337)
- [#5965](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5965) `M9-zigbee-SL` TuYa Smart Switch (4 gang + 4 scene) with neutral wire and motion sensing (@Qianli-Ma)
- [#5967](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5967) `HT-SMO-2` Heimgard Technologies Smoke detector (@sekyHC)
- [#5967](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5967) `HT-DWM-2` Heimgard Technologies Door sensor (@sekyHC)
- [#5964](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5964) `E2201` IKEA RODRET wireless dimmer/power switch (@thatslolo)
- [#5958](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5958) `9290030519` Philips Hue white G93 E27 filament globe (@basschipper)
- [#2081](https://github.com/Koenkk/zigbee2mqtt.io/pull/2081) `A319463` LS Deutschland GmbH Home base (@Koenkk)
- [#5956](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5956) `C210` Candeo Zigbee dimming smart plug (@markxroberts)
- [#5935](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5935) `ZS-TYG3-SM-61Z` TuYa Smart switch (4 gang + 2 scene) with backlight and neutral wire (@Qianli-Ma)
- [#5935](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5935) `ZS-TYG3-SM-21Z` TuYa 2 gang smart switch with backlight and neutral wire (@Qianli-Ma)
- [#5935](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5935) `ZS-TYG3-SM-31Z` TuYa 3 gang smart switch with backlight and neutral wire (@Qianli-Ma)
- [#5935](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5935) `ZS-TYG3-SM-41Z` TuYa 4 gang smart switch with backlight and neutral wire (@Qianli-Ma)
- [#5939](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5939) `MG-ZG01W` TuYa 1 gang switch (@krishnachytanya)
- [#5939](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5939) `MG-ZG02W` TuYa 2 gang switch (@krishnachytanya)
- [#5939](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5939) `MG-ZG03W` TuYa 3 gang switch (@krishnachytanya)
- [#5953](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5953) `TS0003_switch_3_gang_with_backlight` TuYa 3-Gang switch with backlight (@dimich99)
- [#5949](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5949) `B1027EB4Z01` LG Electronics Smart bulb 3 (@luckypants123)
- [#5944](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5944) `3RSPE01044BZ` Third Reality Zigbee / BLE smart plug with power (@hwzolin)
- [#18195](https://github.com/Koenkk/zigbee2mqtt/issues/18195) `PEZ1-042-1020-C1D1` ADEO ENKI LEXMAN Gdansk (@tetienne)

# Fixed device detections
- [#6027](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6027) Detect `_TZE204_ntcy3xu1` as TuYa TS0601_smoke_1 (@wapolinar)
- [#6013](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6013) Detect `_TZ3000_qomxlryd` as ORBIS Windows & Door Sensor, `_TZ3000_qomxlryd` as ORBIS Motion Sensor, `_TZ3000_awvmkayh` as ORBIS Water Sensor and `_TYZB01_821siati` as ORBIS Vibration Sensor (@vsigler)
- [#18434](https://github.com/Koenkk/zigbee2mqtt/issues/18434) Detect `_TZE204_qasjif9e` as TuYa TS0601_smart_human_presence_sensor_1 (@Koenkk)
- [#18418](https://github.com/Koenkk/zigbee2mqtt/issues/18418) Detect `_TZ3210_0aqbrnts` as EFK is-thpl-zb (@Koenkk)
- [#6004](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6004) Detect `_TZ3210_ol1uhvza` as Lonsonho QS-Zigbee-C03 (@rubentest)
- [#6003](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6003) Detect `ZB-SmartPlugIR-1.0.0` as EDP PLUG EDP RE:DY (@dgomes)
- [#5984](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5984) Detect `_TZE200_zxxfv8wi` as HUARUI CMD900LE (@gsergya)
- [#18349](https://github.com/Koenkk/zigbee2mqtt/issues/18349) Detect `_TZ3210_tkkb1ym8` as TuYa TS110E_1gang_2 (@Koenkk)
- [#18355](https://github.com/Koenkk/zigbee2mqtt/discussions/18355) Detect `TAFFETAS2 D1.00P1.03Z1.00` as Acova PERCALE 2 (@Koenkk)
- [#18322](https://github.com/Koenkk/zigbee2mqtt/issues/18322) Detect `EBF_RGB_Zm_CLP` as EGLO 900091 (@Koenkk)
- [#18295](https://github.com/Koenkk/zigbee2mqtt/issues/18295) Detect `_TZ3000_ksw8qtmt` as Nous 1AZ (@Koenkk)
- [#5960](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5960) Detect `_TZE200_eanjj2pa` as Nous SZ-T04 (@Ptr314)
- [#18265](https://github.com/Koenkk/zigbee2mqtt/issues/18265) Detect `_TYZB01_hlla45kx` as ClickSmart+ CMA30036 (@Koenkk)
- [#18242](https://github.com/Koenkk/zigbee2mqtt/issues/18242) Detect `_TYZB01_mtunwanm` as ClickSmart+ CMA30035 (@Koenkk)
- [#18158](https://github.com/Koenkk/zigbee2mqtt/issues/18158) Detect `_TZE200_a8sdabtg` as TuYa ZG-227Z (@Koenkk)
- [#5843](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5843) Detect `_TZE200_feolm6rk` as TuYa TS0601_cover_1 (@1060778506)
- [#13069](https://github.com/Koenkk/zigbee-herdsman-converters/issues/13069) Detect `BSP-GZ2` as Bosch BSP-GZ2 (@stefano-bortolotti)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.32.2'>1.32.2</a>.</em>

**Comment by KoalaWerewolf** [comment]
Is there some way of checking which version of zigbee-herdsman is included in this release?

**Reply by EnsuingRequiem** [reply]
Check out `package.json`
```
    "zigbee-herdsman": "0.17.2", 
    "zigbee-herdsman-converters": "15.50.1",
```

**Comment by kgiedrius** [comment]
Not all devices work after update. Had to rollback to previous version

<img width="1541" alt="Screenshot 2023-08-06 at 09 41 23" src="https://github.com/Koenkk/zigbee2mqtt/assets/15670549/3998e3a4-bc4b-48ad-b526-8b9e079fdcfc">


In previous version works fine brightness adjustment:

<img width="1556" alt="Screenshot 2023-08-06 at 09 26 04" src="https://github.com/Koenkk/zigbee2mqtt/assets/15670549/8cf28bf9-f2ec-465f-8571-995e32a43b92">
